src/test/java/spec/concordion/command/example/ConcordionScopedFieldFixture.java

```diff
@@ -0,0 +1,42 @@
+package spec.concordion.command.example;
+
+import org.concordion.api.ConcordionScopedField;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Created by tim on 2/12/15.
+ */
+@RunWith(ConcordionRunner.class)
+public class ConcordionScopedFieldFixture {
+    @ConcordionScopedField(scope = ConcordionScopedField.Scope.SPECIFICATION)
```

Comment:
I'd suggest we improve the usability of this by creating shorter, focussed annotations, eg. `@SpecificationScoped`.

This would then also be simliar to CDI, which uses scopes such as `@RequestScoped`. See [Scope](https://docs.oracle.com/javaee/6/api/javax/inject/Scope.html) and [Using Producer Methods](https://docs.oracle.com/javaee/6/tutorial/doc/gkgkv.html).